download_and_create_reference_datasets/v02/hail_scripts/write_combined_reference_data_ht.py

```diff
@@ -168,6 +172,67 @@
             'path': 'gs://seqr-reference-data/GRCh38/geno2mp/Geno2MP.variants.liftover_38.ht',
             'select': {'HPO_Count': 'info.HPO_CT'}
         }
+    },
+    'gnomad_mito': {
+        '38': {
+            'path': 'gs://gcp-public-data--gnomad/release/3.1/ht/genomes/gnomad.genomes.v3.1.sites.chrM.ht',
+            'select': ['AN', 'AC_hom', 'AC_het', 'AF_hom', 'AF_het', 'max_hl']
+        }
+    },
+    'mitomap': {
+        '38': {
+            'path': 'gs://seqr-reference-data/GRCh38/mitochondrial/MITOMAP/Mitomap Confirmed Mutations Feb. 04 2022.ht',
+            'select': ['pathogenic']
+        }
+    },
+    'mitimpact': {
+        '38': {
+            'path': 'gs://seqr-reference-data/GRCh38/mitochondrial/MitImpact/MitImpact_db_3.0.7.ht',
+            'select': {'score': 'APOGEE_score'}
+        }
+    },
+    'hmtvar': {
+        '38': {
+            'path': 'gs://seqr-reference-data/GRCh38/mitochondrial/HmtVar/HmtVar%20Jan.%2010%202022.ht',
+            'select': {'score': 'disease_score'}
+        }
+    },
+    'helix_mito': {
+        '38': {
+            'path': 'gs://seqr-reference-data/GRCh38/mitochondrial/Helix/HelixMTdb_20200327.ht',
+            'select': {
+                'AC_hom': 'counts_hom',
+                'AF_hom': 'AF_hom',
+                'AC_het': 'counts_het',
+                'AF_het': 'AF_het',
+                'max_hl': 'max_ARF',
+            }
+        }
+    },
+    'clinvar_mito': {
+        '38': {
+            'path': 'gs://seqr-reference-data/GRCh38/mitochondrial/clinvar/clinvar.GRCh38.chrM.ht',
+            'select': {
+                'allele_id': 'ALLELEID',
+                'clinical_significance': 'CLNSIG',
+                'gold_stars': 'CLNREVSTAT'
+            }
+        }
+    },
+    'dbnsfp_mito': {
```

Comment:
I think it would be better to re-use the SNP dbnsfp configuration. Rather than defining it as a standalone entry here, I think it would be better to do the following after the `CONFIG` definition
```
CONFIG['dbnsfp_mito'] = deepcopy(CONFIG['dbnsfp'] )
CONFIG['dbnsfp_mito']['filter'] = lambda ht: ht.locus.contig == 'chrM'
```